### PR TITLE
Remove last references to DEBUG_SAFESTACK

### PR DESCRIPTION
**Pull request by mspncp**
The DEBUG_SAFESTACK preprocessor define is obsolete since 2008
when the non-safestack code was removed by commit 985de8634000.

### CODE REVIEW

File: Configurations/90-team.norelease.conf [hunk 1]
```diff
@@ -96,7 +96,7 @@ my %targets = (
         cc               => "clang",
         cflags           => combine("-arch x86_64 -DL_ENDIAN",
                                     join(' ', @gcc_devteam_warn),
-                                    "-Wno-error=overlength-strings -Wno-error=extended-offsetof -Wno-error=language-extension-token -Wno-error=unused-const-variable -Wstrict-overflow -Qunused-arguments -DBN_DEBUG -DCONF_DEBUG -DDEBUG_SAFESTACK -DDEBUG_UNUSED -g3 -O3 -pipe",
+                                    "-Wno-error=overlength-strings -Wno-error=extended-offsetof -Wno-error=language-extension-token -Wno-error=unused-const-variable -Wstrict-overflow -Qunused-arguments -DBN_DEBUG -DCONF_DEBUG -g3 -O3 -pipe",
```
Comment (levitte) [inline Configurations/90-team.norelease.conf]: You also removed `DEBUG_UNUSED` here, but not in the "debug-test-64-clang" target.  Please either remove it in all places or restore it here.

Comment (mspncp) [reply Configurations/90-team.norelease.conf]: Ups, I did not notice that in the unified diff. Let me check whether that define is obsolete, too. If it is, then I'll remove it, too.

Comment (mspncp) [reply Configurations/90-team.norelease.conf]: It is still used:

https://github.com/openssl/openssl/blob/bf5b04ea25d6ac7d31e388b4e87d3eb5cd1e1e2b/include/openssl/e_os2.h#L224-L228

Comment (mspncp) [reply Configurations/90-team.norelease.conf]: Fixed in f53845b9081837ca4aa669b5318791d0baa4df8f.

**Comment by mspncp** [comment]
Merged, thank you!

a345ce6cb1 Remove last references to DEBUG_SAFESTACK [1.1.1]
d4f63f1c39 Remove last references to DEBUG_SAFESTACK [master]

**Pull request closed without merging**